assets/scripts/segments/variant_icons.json

```diff
@@ -280,6 +280,16 @@
       "title": "Perpendicular parking, road level"
     }
   },
+  "bike-lane-elevation": {
```

Comment:
I'm curious if this works by using `elevation` instead of defining a new `bike-lane-elevation` variant. We have a generic `elevation` defined already. Segments with elevation settings had their own defined variants previously, but this might not necessarily be a great idea moving forward if anything can potentially be on a sidewalk or not.

Reply:
I noticed the "elevation" property on components, but it doesn't seem to be referenced by anything. It's also not a variant, so I'm not sure how it would be used to change the property. We could certainly rename this variant (and the one for scooter lanes) to just "elevation."

Reply:
I'd propose renaming to "elevation" and see if it works right now.

Reply:
It works fine still just called 'elevation.' I pushed a fix to do so. Should we be renaming 'bike-rack-elevation', 'scooter-elevation', 'transit-shelter-elevation', etc. also?

Reply:
I held back on those ones for now as they would need a schema update since they're already on prod.

Reply:
Yep. I'll create an issue to follow up on this. We don't need to blow up the scope of this one.